Pipfile

```diff
@@ -4,15 +4,14 @@ url = "https://pypi.org/simple"
 verify_ssl = true
 
 [dev-packages]
-pylint = "*"
-pytest = "*"
-pytest-mock = "*"
-pre-commit = "*"
-
-[packages]
-aws-sam-cli = "*"
-boto3 = "*"
-botocore = "*"
+aws-sam-cli = "~=1.2.0"
+boto3 = "~=1.14.58"
```

Comment:
need to add awscli to this list and we don't need boto3 or botocore because those will get brought in by awscli & aws-sam-cli

Reply:
fixed